orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -100,14 +100,18 @@ class DependentPipelineStarter implements ApplicationContextAware {
 
     def trigger = pipelineConfig.trigger
     //keep the trigger as the preprocessor removes it.
-    def expectedArtifacts = pipelineConfig.expectedArtifacts
```

Comment:
It looks like this is reverting the fix that was made in spinnaker/orca#2430.  Have you tested that you haven't re-introduced the bug that was fixed there?